dev/.env

```diff
@@ -0,0 +1,3 @@
+POSTGRES_URI=postgresql+psycopg://nautilus:nautilus@database:5432/nautilus
+CHANNEL_NAME=s3_upload
+REDIS_URI=redis://redis:6379/0
```

Comment:
S3 credentials is also needed

Reply:
No, We shouldn't put S3 credentials here. S3 credentials is a private key. The most we can put here is a placeholder

Reply:
That's what I mean, a placeholder.